### DISCUSSION TITLE
Where can I find samples of using Ky?

### DISCUSSION THREAD

**Discussion by TechWatching** [sindresorhus/ky#427] [Q&A] [answered]
I am looking for examples of open source applications (vuejs 3 would be great by actually any application would do) using Ky. I want to know more about good practices of organizing the code using Ky, how people use it in their application and not just on a 2 lignes example.

Would you happen to know where I can find such samples? Thanks in advance.

**Comment by sindresorhus** [comment, marked as answer]
I don't have any specific example, but you could try taking a look [here](https://github.com/search?q=%22import+ky+from%22&type=code) and [here](https://github.com/sindresorhus/ky/network/dependents).

**Reply by TechWatching** [reply]
Good idea. I will give it a look. Thanks.

**Reply by uchihamalolan** [reply]
You could also try the new [Github Code Search](https://cs.github.com/?q=%22import%20ky%20from%22%20NOT%20language%3AMarkdown&scopeName=All%20repos&scope=) here

**Reply by TechWatching** [reply]
Yes that is what I finally did..
